### ISSUE TITLE
Problems with IreadMail

### ISSUE THREAD

**Issue by rgbringas** [iredmail/iRedMail#179]
Hello, I have updated the version of my iRedMail (1.5.2), I have two domains configured on my server in Ubuntu Server 20.04 but after updating the server the iRedMail website does not open
gives me the error:
**internal server error**
The webmail of the two email domains open perfectly for me, however, I send emails that do not reach any address that I have tested (nor to email internally) although Roundcube does not give an error.
I also do not receive any email externally.
Any idea about it????

**Comment by iredmail** [comment]
Any related error in /var/log/syslog?

**Comment by rgbringas** [comment]
Hi, I ended up doing a fresh new install yesterday. At the end of the day I can access ireadmin (iRedMail) and Rounduce with my two domains, only details remained.
Initially Roundcube gave an SMTP error
**SMTP ERROR: (-1) Failed to connect to server**
I updated the port to 465, and now I get the error:
**SMTP Error (454) Authentication Failed**
I look at everything, I have gone through step by step, everything seems to be fine.

Any idea about it?
